packages/plans-grid/src/plans-table/plan-item.tsx

```diff
@@ -81,8 +87,14 @@ const PlanItem: React.FunctionComponent< Props > = ( {
 	const isOpen = allPlansExpanded || isDesktop || isPopular || isOpenInternalState;
 
 	return (
-		<div className={ classNames( 'plan-item', { 'is-popular': isPopular, 'is-open': isOpen } ) }>
-			{ isPopular && (
+		<div
+			className={ classNames( 'plan-item', {
+				'is-popular': isPopular,
+				'is-open': isOpen,
+				'badge-next-to-name': popularBadgeVariation === 'NEXT_TO_NAME',
+			} ) }
+		>
+			{ isPopular && popularBadgeVariation === 'ON_TOP' && (
```

Comment:
If we export the `popularBadgeVariation ` values, we can use them here instead of hard coding `'ON_TOP'`

Reply:
I think exporting the type is enough. The IDE won't let you set an invalid value.